lib/src/components/personSearch/PersonSearch.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { DropdownContent, Input, Label, Person } from 'lib';
+import Dropdown from '../../../Dropdown';
+
+export function PersonSearch({ people, onPersonSelect, label, placeholder }) {
+  const [searchValue, setSearchValue] = useState('');
+
+  const filteredPeople = searchValue
+    ? people.filter(p =>
+        p.name.toLowerCase().includes(searchValue.toLowerCase())
+      )
+    : people;
+
+  return (
+    <>
+      <Dropdown id="person-search-story" className="overflow-visible w-full">
+        {({ showContent }) => (
+          <>
+            <Dropdown.Trigger triggerClassName="w-full">
+              {({ toggleShowContent }) => (
+                <>
+                  <Label htmlFor="person-search">{label}</Label>
+                  <Input
+                    id="person-search"
+                    placeholder={placeholder}
+                    onFocus={() => toggleShowContent(true)}
+                    onBlur={() => toggleShowContent(true)}
+                    onChange={e => setSearchValue(e.target.value)}
+                    value={searchValue}
+                  />
+                </>
+              )}
+            </Dropdown.Trigger>
+            {showContent && (
+              <Dropdown.Content noTransform collapse className="w-full">
+                <DropdownContent>
```

Comment:
💚 This could be split up a bit. There’s a few nested ternarys here that makes it hard to parse. 

Maybe something like `PersonSearchContent` `PersonSearchNoResults` `PersonSearchResults`?